src/components/Sum/Sum.jsx

```diff
@@ -1 +1,12 @@
-// export const Sum = () => ();
+export const Sum = ({ a = 0, b = 0 }) => (
+  <p>
+    Sum of
+    {a}
+    {' '}
+    and
+    {b}
+    {' '}
+    is
+    {a + b}
+  </p>
```

Comment:
```suggestion
  <p>
    {`Sum of ${a} and ${b} is ${a + b}`}
```
use template string here